en/docs/administer/configuring-a-jdbc-user-store.md

```diff
@@ -1,163 +1,174 @@
 # Configuring a JDBC User Store
 
 User management functionality is provided by default in all WSO2
-Carbon-based products and is configured in the user-mgt.xml file found
-in the \<PRODUCT\_HOME\>/repository/conf/ directory. This file is
-shipped with user store manager configurations for all possible user
-store types (JDBC, read-only LDAP/Active Directory, read-write LDAP and
-read-write Active directory). The instructions given below explains how
-to configure an RDBMS (JDBC) as the primary user store for the WSO2
-server.
+Carbon-based products and is configured in the
+`         deployment.toml       ` file found in the
+`         <IS_HOME>/repository/conf/        ` directory. This
+documentation explains the process of setting up a primary user store
+for your system.
 
 !!! info "The default User Store"
-    The internal H2 database that is shipped with every WSO2 product (except
-    WSO2 Identity Server) is configured as the default primary user store.
-    This internal database is used by the Authorization Manager (for user
-    authentication information) as well as the User Store Manager (for
-    defining users and roles). In the case of the WSO2 Identity Server, the
-    default user store is an LDAP (Apache DS) that is shipped with the
-    product.
-
-When you configure a JDBC user store as the primary user store, you can
-either use the default configuration or you can change it in the
-following ways:
-
--   You can set up two separate databases for the Authorization Manager
-    and the User Store Manager.
--   It is not recommended to use the default H2 database in production.
-    Therefore, you can replace this. For instructions on replacing this
-    by setting up a new RDBMS and configuring it for your system, see
-    [Setting Up the Physical
-    Database](../../administer/working-with-databases).
-
-Therefore, before you begin, ensure that the RDBMS that you want to use
-as the JDBC user store is correctly set up for your system. Then, follow
-the steps given below to configure a JDBC user store as the primary user
-store in your product.
-
-### Step 1: Configuring the JDBC user store manager
-
-
-To configure a JDBC user store as the primary user store, you must
+    The primary user store that is configured by default in WSO2 Identity Server
+    is an LDAP user store, This database is used by the Authorization Manager (for user authentication
+    information) as well as the User Store Manager (for defining users and
+    roles).
+
+We can specify the type of user store that we are using in the `         deployment.toml       ` file. 
```

Comment:
You can specify the type of user store that you are using in the `deployment.toml` file. If it is specified as a JDBC user store by default, see [Working with Databases](../../administer/working-with-databases) for a detailed guide on configuring an RDBMS database as a datasource.